KMeansUtil.py

```diff
@@ -0,0 +1,81 @@
+import math, utils, random
+
+class KMeansUtil:
+
+	#Constructor to initialize the value of K
+	def __init__(self):
+		self.k = int(input("Enter the number of clusters: "))
+
+
+	#A method to initialize all K clusters to empty lists
+	def initialize(self, cluster, dist):
+		for i in range(self.k):
+			clusterList = []
+			cluster.append(clusterList)
+			dist.append(0)
+
+		return cluster, dist
+
+
+	#A method for generating K random centroids
+	def genRandomCentroids(self, centroid, oldCentroid):
+		for c in range(self.k):
+			centroidx = random.randrange(4600, 5100)
```

Comment:
Externalize the constants please.

Reply:
I have done this using .json. I will send a PR today.